README.md

```diff
@@ -515,6 +515,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Apache FOP](http://xmlgraphics.apache.org/fop/) - Creates PDF from XSL-FO.
 * [Apache PDFBox](http://pdfbox.apache.org/) - Toolbox for creating and manipulating PDF.
 * [DynamicReports](http://dynamicreports.org/) - Simplifies JasperReports.
+* [Dyanmic Jasper](http://dynamicjasper.com/) -  hides the complexity of JasperReports, it helps developers to save time when designing simple/medium complexity reports generating the layout of the report elements automatically. It creates reports dynamically, defining at runtime the columns, column width (auto width), groups, variables, fonts, charts, crosstabs, sub reports (that can also be dynamic), page size and everything else that you can define at design time.
```

Comment:
Please write a more condensed description and use the same format as the other entries.